openid-4-verifiable-presentations-1_0.md

```diff
@@ -1537,34 +1537,11 @@ And lastly, as part of the request, the Wallet is provided with information abou
 
 ## Protocol
 
-The value of the `protocol` parameter of the W3C Digital Credentials API MUST be set to `openid4vp` for this profile.
+When used with the W3C Digital Credentials API [@!w3c.digital_credentials_api], the exchange protocol is the string `openid4vp` for this profile.
 
 ## Request {#browser_api_request}
 
-The `request` member of the W3C Digital Credentials API [@!w3c.digital_credentials_api] contains an OpenID4VP Authorization Request, where every OpenID4VP Authorization Request parameter is represented as a top-level JavaScript object member.
-
-The following is a non-normative example of how the W3C Digital Credentials API can be used with an unsigned OpenID4VP request when advanced security features of OpenID4VP are not used:
-
-```js
-try {
-  const credential = await navigator.identity.get({
-    digital: {
-      providers: [{
-        protocol: "openid4vp",
-        request:  {
-          response_type: "vp_token",
-          response_mode: "w3c_dc_api",
-          nonce: "n-0S6_WzA2Mj",
-          client_metadata: {...},
-          presentation_definition: {...}
-        }
-      }]
-    }
-  });
-} catch (err) {
-  // Handle errors and/or fallback to other invocation mechanisms
-}
-```
```

Comment:
I can't figure out how to actually make a suggestion that undoes the delete, but I believe we should NOT remove the actual OID4VP part of the examples. It was clear from last night's DCP WG call that many people don't understand what is passed by OID4VP to the browser API, and removing the examples will definitely make that even harder to figure out.

I hence suggest we retain this part of the example:

```
{
           response_type: "vp_token",
           response_mode: "w3c_dc_api",
           nonce: "n-0S6_WzA2Mj",
           client_metadata: {...},
           presentation_definition: {...}
         }
```

(and apply a similar change to other removed examples)

Reply:
Strongly agree

Reply:
Discussed on today's WG call, consensus to keeping the payload parts but removing the rest of the example.

Reply:
Should be good now. @jogu, @Sakurann, can you confirm.